### PR TITLE
cellGcm HLE + sys_usbd: Add error_code

### PR DESCRIPTION
**Pull request by elad335**

### CODE REVIEW

File: rpcs3/Emu/Cell/lv2/sys_usbd.cpp [hunk 1]
```diff
@@ -498,7 +498,7 @@ s32 sys_usbd_get_device_list(u32 handle, vm::ptr<UsbInternalDevice> device_list,
 	return i_tocopy;
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/lv2/sys_usbd.cpp]: !

File: rpcs3/Emu/Cell/lv2/sys_usbd.cpp [hunk 2]
```diff
@@ -515,7 +515,7 @@ s32 sys_usbd_register_extra_ldd(u32 handle, vm::ptr<char> s_product, u16 slen_pr
 	return res; // To check
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/lv2/sys_usbd.cpp]: !

**Comment by elad335** [comment]
Review comments addressed, btw cellGcmGetDisplayBufferByFlipIndex doesnt have error code indeed.

**Comment by Megamouse** [comment]
Can't review the extra stuff so :man_shrugging:

**Pull request merged**